config/_default/params.toml

```diff
@@ -30,6 +30,8 @@ highlight = true
 # highlight_languages = ["r"]  # Add support for highlighting additional languages
 highlight_style = "github"  # For supported styles, see https://cdnjs.com/libraries/highlight.js/
 
+highlight_languages = ["yaml", "shell"]
```

Comment:
We have many other languages like `bash`, `toml`，now seems partly highlighted, part not highlighted.. Maybe we need unify them?

Reply:
The highlight_languages configuration can move to another PR, let's fix the docs first :)

Reply:
OK, I will move the change of `highlight` to another pr~